### PR TITLE
Feature/header secondary variant

### PR DESCRIPTION
**Pull request by stevenbister**
### 🧹 Please check off the following before submitting:

- [ ] Close issues
- [x] Request reviewers
- [x] Assign yourself
- [ ] Add labels (e.g. `build:osc-ecommerce`)
- [x] Add project
- [x] Set project status
- [ ] Add milestone

---

## ☑ List all github issues that this PR will close:

N/a

## 📝 Description

Adds the secondary variant of the header to build into the academic hub app shell
image

## ⛳️ Current behaviour (updates)

N/a

## 🚀 New behaviour

- Adds secondary header variant

## 💣 Is this a breaking change (Yes/No): No

> Please describe any breaking changes.

## 🧾 Final Checklist

- [ ] I've considered possible cross-platform impacts (Mac, Linux, Windows)
- [x] I've considered possible cross-device impacts (iOS, Android)
- [x] I've considered possible cross-browser impacts (Chrome, Safari, Edge, Firefox)
- [ ] I've added tests to cover my changes (Unit, E2E, Integration, Contract, Functional, Acceptance etc)
- [ ] I've added or updated any documentation.

## 📝 Additional Information

> Any additional contextual information, such as relevant links/articles/documentation

### CODE REVIEW

File: packages/osc-ui/src/components/Header/header.scss [hunk 1]
```diff
@@ -162,6 +172,12 @@
 
                     flex-shrink: 0;
                 }
+
+                &--secondary {
+                    .o-icon {
+                        --icon-size: 23px;
+                    }
+                }
```
Comment (jkellerman) [inline packages/osc-ui/src/components/Header/header.scss:183]: In the design, the icon in mobile is 31px, should we have media query here?

Comment (jkellerman) [reply packages/osc-ui/src/components/Header/header.scss:183]: Just saw you have separate styles for avatar, I'm assuming this is taken care of here?

**Review by jkellerman — COMMENTED** [review]
Great! Just a few more questions. 
1. How come the background colour for the icon is using a gradient when the background in the design is blue?
2. Shouldn't we include some kind of hover state for icon/avatar and the nav buttons?

File: packages/osc-ui/src/components/Header/header.scss (hunk 1 above)
Comment (stevenbister) [reply packages/osc-ui/src/components/Header/header.scss:183]: Yeah this style doesn't control the avatar it's for the icons. The avatar styles are handled by the `.header__avatar`class

**Comment by stevenbister** [comment]
Thanks man! To answer your questions

> 1. How come the background colour for the icon is using a gradient when the background in the design is blue?

I'm assuming you're talking about the avatar? The design is just a guide in this case, the avatar component has a function in it that generates a background colour based on the name passed into it. I think it was a decision we made a while ago when we built the component to do it this way

> 2. Shouldn't we include some kind of hover state for icon/avatar and the nav buttons?

I don't think the avatar needs a hover state but yeah should have wrapped the icons in the `<Button>` component

**Review by jkellerman — APPROVED** [review]
LGTM

**Pull request merged**